core/src/stored/stored.cc

```diff
@@ -179,10 +181,14 @@ int main(int argc, char* argv[])
         prt_kaboom = true;
         break;
 
-      case 'p': /* proceed in spite of I/O errors */
+      case 'k': /* proceed in spite of I/O errors */
```

Comment:
Now I see that you have switched to "-k". I think "-i" would be better. Also, this change needs to be added to the --help output.